### Using the Monte carlo simulation to check If ballistics backlog is 365 days and what is the probability a murder case gets delayed by more 2 years?

In [3]:
import numpy as np
backlog = np.random.normal(365, 90, 10000) # mean 365, std 90 days and 10000 as an example
court_delay = np.random.normal(180, 30, 10000)
total_delay = backlog + court_delay
print(f"Probability of delay > 730 days: {(total_delay > 700).mean()*100:.1f}%") # 730 days is 2 years

Probability of delay > 730 days: 5.3%


 ### The 5.3% on my output is very low and not realistic for South African murder cases
 ### My std deviation is low, our SA backlog is not normal because we have cases that have no progress report up to a year
 ### I did not consider delays in laboratries and evidence lost at stations